Octokit/Http/IHttpClient.cs

```diff
@@ -10,7 +10,7 @@ namespace Octokit.Internal
     /// <remarks>
     /// Most folks won't ever need to swap this out. But if you're trying to run this on Windows Phone, you might.
     /// </remarks>
-    public interface IHttpClient : IDisposable
+    public interface IHttpClient : IDisposable, IApiInfo
```

Comment:
This does not feel right. API info is not a HTTP concern (see previous comment).